pkg/integrations/manager.go

```diff
@@ -220,9 +222,14 @@ func (m *Manager) ApplyConfig(cfg ManagerConfig) error {
 				continue
 			}
 			p.stop()
+			m.im.DeleteConfig(key)
```

Comment:
Beforehand the scraper was not stopping, but the integration was.

Reply:
I'm not following how this is necessary, the second loop should be deleting the config?

Reply:
With the addition of the enabled check around line 227 the integration process is not started so the integration isn't added to the to m.integrations.